src/screens/SearchScreen.js

```diff
@@ -3,49 +3,40 @@ import {View, Text, ActivityIndicator, StyleSheet, Platform, Keyboard} from 'rea
 import {Button} from 'react-native-elements';
 import {MapView} from 'expo';
 import {Icon} from 'react-native-elements';
+import {connect} from 'react-redux';
+
+import {initializeMap, updateViewStyle, mapLoaded, updateRegion} from "../actions";
 
 import _ from 'lodash';
 
 import {CardSection, Card} from '../components/common/index';
 
-const SearchOptions = {
-    MAP: 'map',
-    LIST: 'list'
-};
+import {map} from '../constants';
```

Comment:
No longer declaring constants in the file, and now pulling them from the central `constants` file, since we'll use them in several places.